src/main/resources/db/migration/V1.0.3__create_obs_unit_sequence.sql

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-alter table program add column obs_unit_sequence text;
+alter table program add column  IF NOT EXISTS obs_unit_sequence text;
```

Comment:
Just stating this so it's out of my head.  Modifying these existing scripts will mean the deploy to QA will fail, because we'll have to modify the `flyway_schema_history` table to reflect the new checksum this file will produce

Reply:
Done